cypress/e2e/tests/migration/controls/businessservices/crud.test.ts

```diff
@@ -77,24 +92,34 @@ describe(["@tier1", "@interop"], "Business service CRUD operations", () => {
             stakeholder.name
         );
         businessService.create();
+        checkSuccessAlert(
+            commonView.successAlertMessage,
+            `Success alert:Business service ${businessService.name} was successfully created.`
+        );
         cy.wait("@postBusinessService");
         exists(businessService.name);
 
         // Edit Business service's name
         let updatedBusinessServiceName = data.getCompanyName();
         businessService.edit({ name: updatedBusinessServiceName });
+        checkSuccessAlert(
+            commonView.successAlertMessage,
+            `Success alert:Business service ${businessService.name} was successfully saved.`
+        );
         cy.wait("@getBusinessService");
         exists(updatedBusinessServiceName);
 
         // Delete Business service
         businessService.delete();
+        checkSuccessAlert(commonView.successAlertMessage, `Success alert:Business service deleted`);
         cy.wait("@getBusinessService");
 
         // Assert that Business service is deleted
         notExists(businessService.name);
 
         // Delete stakeholder owner
         stakeholder.delete();
+        checkSuccessAlert(commonView.successAlertMessage, `Success alert:Business service deleted`);
```

Comment:
Here we're deleting a stakeholder, not a business service, right? The notification message isn't the correct one.